### ISSUE TITLE
Vivado does not use the RTL gates when creating EDIF files

### ISSUE THREAD

**Issue by ghost** [RohanNagar/parallel-logic-networks#1]
Hello,

In all of your Vivado .edn and .edf file examples your files contain discrete gate types like RTL_ADD, RTL_XOR, and RTL_0R0. However when I attempt to write_edif file I get primitives like LUT2, and LUT3 which are defined as truth tables in the design. I am wondering if there was a trick that you all did to get the EDIF files to output using discrete gates. 

I am using the same version of vivado that appears on your files

Thank you

**Comment by aytung94** [comment]
Hi,

There was nothing specific we did to get the gate netlist to have only discrete gates. The reason being is that we tested very simple behavioral defined and gate level defined combinaton circuits. The LUT or look up table is a logic block specific to the target architecture that maps the logic to hardware during implementation process. In our case, we selected the Atrix-7 FPGA by Xilinx.

This link might be of use: https://forums.xilinx.com/t5/Synthesis/Generating-Gate-level-netlist-with-discrete-gates/td-p/44841
